scripts/build_db_image

```diff
@@ -11,7 +11,7 @@ tag="${ecr_backend}:${3:-${CIRCLE_SHA1}}"
 builddir="$(git rev-parse --show-toplevel)"
 
 # log in to ECR
-if bash +x -o nounset -c "$(aws ecr get-login --no-include-email --region "us-west-2" --registry-ids "${AWS_ACCOUNT_ID}")" ; then
+if (set +x -o nounset; aws ecr get-login-password --region "${AWS_DEFAULT_REGION}" | docker login --username AWS --password-stdin "${AWS_ACCOUNT_ID}".dkr.ecr."${AWS_DEFAULT_REGION}".amazonaws.com) ; then
```

Comment:
Reading passwords from stdin is the way to go. Thanks for this.

Curious, why did you prefer the subshell to `bash -c` ?

Reply:
`get-login`, which is deprecated, prints a docker login command to be executed. Piping the `get-login-password` output directly to the `docker login` command avoids the need for `bash -c`. I initially kept the `bash -c` but got the following [error](https://circleci.west.cms.gov/gh/CMSgov/easi-app/9201): `bash: Login: command not found`

Reply:
Makes sense, thanks